core/experiments/manager.php

```diff
@@ -455,6 +455,14 @@ private function add_default_features() {
 				'container',
 			],
 		] );
+
+		$this->add_feature( [
+			'name' => 'e_optimized_control_loading',
+			'title' => esc_html__( 'Optimized Control Loading', 'elementor' ),
+			'tag' => esc_html__( 'Performance', 'elementor' ),
+			'release_status' => self::RELEASE_STATUS_BETA,
+			'default' => self::STATE_INACTIVE,
+		] );
```

Comment:
maybe it needs to be in the frontend performance module?
not sure about it though